src/common/internal/test_group.ts

```diff
@@ -126,6 +126,12 @@ interface TestBuilderWithName<F extends Fixture> extends TestBuilderWithParams<F
    * @param id a token uniquely assigned to this test.
    */
   uniqueId(id: string): this;
+  /**
+   * A noop function with the purpose of highlighting value `url`.
```

Comment:
`highlighting value` is a bit non-descriptive.

Its purpose is to associate a CTS test with the relevant part of the specification.